srcpkgs/newsboat/template

```diff
@@ -25,8 +25,4 @@ do_check() {
 
 post_install() {
 	vlicense LICENSE
-	vsconf doc/example-bookmark-plugin.sh bookmark-plugin.sh
```

Comment:
please double-check the lines you removed from `post_install`,
for example: `diff <(xbps-query -Rf newsboat) <(fd -t f . usr | sort | sed 's/^/\//')`

Reply:
This file is already at `/usr/share/doc/newsboat/examples/example-bookmark-plugin.sh`.
I made sure that no actual files were lost.

Reply:
i dind't mean that file in specific but the others. my diff has about 20 changed lines. please run my snippet (in `destdir/newsboat-...`)

Reply:
I diffed the package contents. The `contrib` folder is currently present twice in the package, once installed by `make install` to `/usr/share/doc/newsboat/contrib` and then again by the `vcopy` to `/usr/share/examples/newsboat/contrib`. That's I was trying to fix.